app/util/planParamUtil.js

```diff
@@ -296,6 +323,10 @@ export function getPlanParams(
     case PLANTYPE.TRANSIT:
       direct = access;
       break;
+    case PLANTYPE.SCOOTERTRANSIT:
+      access = ['WALK', 'SCOOTER_RENTAL'];
+      egress = access;
+      break;
```

Comment:
One more change: set numItineraries to 1 if searching scooter.

Reply:
The search also returns plain scooter itineraries, which are not shown on the UI. Searching for more than one itinerary gives a better chance to receive an itinerary with another mode of transport. It's not very efficient but I don't think it's possible to force the query to not return plain scooter journeys.

Reply:
Maybe directOnly=false option will help?

Reply:
Actually, now that I checked again, the problem was that we can't force the itinerary to contain a scooter journey so it's better to search more so that at least one of the itineraries contains a scooter leg as well if it's feasible.

Reply:
Right, that indeed makes sense.